index.rst

```diff
@@ -46,14 +46,314 @@
+This operation mode has been used with the Vera Rubin Observatory system from the beginning and, despite some occasional issues, worked relatively well over time.
+In fact, most of the time we encountered issues with the DDS communication we could relate them to one of the following issues:
+
+  * Wrong DDS partition.
+
+    The DDS partition is controlled by SAL and SalObj by the ``LSST_DDS_DOMAIN`` system environment variable.
+    It was common on some occasions that users would set different values for that variable, which causes DDS to filter the messages and communication would not work.
+    The solution is to make sure everyone on the system has the same values for the environment variable.
+
+  * Systems running incompatible versions of xml interface.
+
+    The xml interface determines the DDS topics data structure.
+    If topics with the same identity have different data structures the last system to try to connect to the DDS network will fail.
+    This created several issues in the past as DDS cache would prevent users from upgrading the topic data structure.
+    A mitigation to this problem was introduced in recent versions of SAL where topics identity was attached to a hash code based on its content.
+    This allowed users to upgrade topics without colliding with previous cached versions but also meant that incompatible versions silently fail to communicate.
+
+  * Systems running with incompatible versions of SAL/SalObj.
+
+    SAL and SalObj are responsible for managing topics Quality of Service (QoS) parameters.
+    The system defines 4 different types of QoS depending on the "type" of topic: Command, Event, Telemetry and Acknowledgement.
+    Topics with different QoS may sometimes be filtered out by the DDS reader so, an incompatible version of SAL/SalObj may not be able to exchange some messages.
+    This was improved considerably in more recent versions of SAL/SalObj and we have been able to keep breakages to a minimum, especially by rigidly managing deployments.
+
+  * Wrong network attached.
+
+    By default the OpenSplice configuration will automatically select a network to attach to.
+    It mainly works by selecting the first suitable network (excluding local and loopback networks for instance) in alphabetical order.
+    For servers with more than one "suitable network" it may happen that the selected network is not the correct one.
+    In this cases, users must edit the configuration file and explicitly select network attached to where the DDS communication is supposed to happen.
+
+As we started to operate the system with an increased number of components, and especially, components with large sets of topics published at higher frequencies (20 to 50 Hz, as opposed to 1Hz) we started to observe a new failure mode.
+This failure would, in principle, happened at random and is mostly characterized by long or timeouts in reading historical data followed by unreliable DDS communication.
```

Comment:
would ?? This isn't conditional is it?

"This failure, in principle, happens at random...."